public/app/features/explore/RichHistory/RichHistoryQueriesTab.tsx

```diff
@@ -31,8 +31,10 @@ export interface Props {
   retentionPeriod: number;
   exploreId: ExploreId;
   height: number;
+  searchFilter: string;
   onChangeSortOrder: (sortOrder: SortOrder) => void;
   onSelectDatasourceFilters: (value: SelectableValue[] | null) => void;
+  onSearchQuery: (value: string) => void;
```

Comment:
I'd call this `onSearchFilterChange`, to be consistent with `searchFilter` and it being a change event.

Reply:
Updated in https://github.com/grafana/grafana/pull/25747/commits/3313c6b0e430ceeb85beb17670c8b1215518229a